run.py

```diff
@@ -3,8 +3,18 @@
 from block_io import BlockIo
 import os
 import math
+import json
+from github import Github
```

Comment:
We have some new requirements coming in here that we would need to add to the requirements file at `Pipfile`. Updated `Pipfile` could look something like:

```
[requires]
python_version = "3.6"

[packages]
block-io = "*"
PyGithub = "*"
PyYAML = "*"

```

Reply:
ok done